website/blog/2018-06-19-single-spa-parcels-explained.md

```diff
@@ -0,0 +1,70 @@
+---
+title: single-spa parcels, explained
+author: Joel Denning
+authorURL: https://twitter.com/joelbdenning
+authorImageURL: https://avatars2.githubusercontent.com/u/5524384?s=400&u=ff145fcb2ae5305555628a446e9f725d4e145aaa&v=4
```

Comment:
Does this work for you? I haven't been able to figure out how to get the photo to actually render.

Reply:
no it doesn't do anything afaict. We'd have to dive into https://docusaurus.io/docs/en/blog to figure it out I think